src/test/java/net/minecraftforge/debug/ClientExceptionTestMod.java

```diff
@@ -0,0 +1,61 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiErrorScreen;
+import net.minecraftforge.fml.client.CustomModLoadingErrorDisplayException;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+@Mod(modid = "clientexceptiontest", version = "0", name = "Client Exception Test")
+public class ClientExceptionTestMod
+{
+
+    // Disabled so other test mods can still work.
+    public static boolean ENABLE_PREINIT = false, ENABLE_INIT = false;
+    private static String LOREM_IPSUM = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.";
+
+    @Mod.EventHandler
+    @SideOnly(Side.CLIENT)
```

Comment:
Client ones are good, in general. Server ones often cause crashes when it is on the client.